gatsby-config.js

```diff
@@ -23,9 +25,18 @@ module.exports = {
         includeCollections: ["shop", "content"],
       },
     },
+    {
+      resolve: "gatsby-plugin-google-analytics",
+      options: {
+        // replace "UA-XXXXXXXXX-X" with your own Tracking ID
+        trackingId: "UA-XXXXXXXXX-X",
+      },
+    },
   ],
   siteMetadata: {
     title: "Netlify Swag Store",
     description: "Default meta description",
```

Comment:
```suggestion
    description: "Netlify socks, stickers, shirts, mugs, and much more! Check out the store for official Netliswag and Jamstack gear. PS - plz share with other devs.",
```

definitely thinking we can have some fun here, and that devs will appreciate something like this :-) 
/cc @lsell @tabar thoughts on description? will appear in twitter, etc.